examples/lb/members/base-image.tf

```diff
@@ -0,0 +1,3 @@
+data "vkcs_images_image" "ubuntu18-04" {
```

Comment:
why do not link to image datasource example?

Reply:
here we may use images/image/datasource